generate_and_apply_patch.sh

```diff
@@ -42,20 +54,37 @@ cp dsdt.aml kernel/firmware/acpi
 find kernel | cpio -H newc --create > acpi_override
 
 # copy override file to boot partition
-sudo cp acpi_override ${BOOT} || \
-	{ echo "ERROR: Could not copy acpi_override"; exit $?; }
-
-# check if we have initramfs and prepend our stuff
-if [ -d $(dirname "${IRFS_HOOK}") ] && [ -x $(which update-initramfs) ]; then
-	echo "[*] Adding hook to initramfs"
-	sudo bash -c "cat > ${IRFS_HOOK} <<- HOOK
-	#!/bin/sh
-	. /usr/share/initramfs-tools/hook-functions
-	prepend_earlyinitramfs /boot/acpi_override
-	HOOK"
-	sudo chmod +x ${IRFS_HOOK}
-	sudo update-initramfs -u -k all
-	echo "[*] Done!"
-else
-    echo "Done! Don't forget to update your bootloader config."
-fi
+sudo cp acpi_override ${BOOT}/acpi_override.img || \
```

Comment:
This was named `acpi_override` according to https://wiki.archlinux.org/index.php/DSDT#Using_a_CPIO_archive

I like the detailed explanation in the [Microcode](https://wiki.archlinux.org/index.php/Microcode) article.

However, this potentially breaks upgrading for people that use `acpi_override` instead of `acpi_override.img`.

Could you consider dropping the renaming here?